error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/ReactorRulesTestInput.java

```diff
@@ -639,6 +639,36 @@ void testPublisherProbeAssertWasNotRequested() {
     assertThat(PublisherProbe.empty().wasRequested()).isFalse();
   }
 
+  void testPublisherProbeAssertThatWasSubscribed() {
+    PublisherProbe<Integer> probe = PublisherProbe.of(Mono.just(1));
+    boolean wasSubscribed = true;
+    if (wasSubscribed) {
+      probe.assertWasSubscribed();
+    } else {
+      probe.assertWasNotSubscribed();
+    }
+  }
```

Comment:
Given that the Refaster rule doesn't return anything, I frankly don't know how to test both `true` and `false` cases. So for now, I only tested one of them.

Reply:
Obvious once you see it, thanks!